tests/unit/lms/services/lti_h_test.py

```diff
@@ -68,23 +68,6 @@ def test_it_raises_if_creating_the_group_fails(
         with pytest.raises(HTTPInternalServerError, match="Oops"):
             lti_h_svc.upsert_course_group()
 
-    def test_it_400s_with_missing_group_and_unpriviledged_user(
-        self, pyramid_request, h_api, lti_h_svc
-    ):
-        pyramid_request.lti_user = LTIUser(
-            user_id="TEST_USER_ID",
-            oauth_consumer_key="TEST_OAUTH_CONSUMER_KEY",
-            roles="learner",
-        )
-        h_api.update_group.side_effect = HAPINotFoundError()
-
-        with pytest.raises(
-            HTTPBadRequest, match="Instructor must launch assignment first"
-        ):
-            lti_h_svc.upsert_course_group()
-
-        h_api.create_group.assert_not_called()
```

Comment:
We used to show an error page if the group didn't exist in h yet and the user wasn't a teacher. I'm not sure how often that actually happened in practice, but the code is simpler now as the error case has been removed.